pkg/kubecost/status.go

```diff
@@ -33,3 +33,31 @@ type FileStatus struct {
 	Errors       []string          `json:"errors,omitempty"`
 	Warnings     []string          `json:"warnings,omitempty"`
 }
+
+// CloudStatus describes CloudStore metadata
+type CloudStatus struct {
+	CloudAssets    *CloudAssetStatus     `json:"cloudAssets,omitempty`
+	Reconciliation *ReconciliationStatus `json:"reconciliation,omitempty"`
+}
+
+// CloudAssetStatus describes CloudAsset metadata of a CloudStore
+type CloudAssetStatus struct {
+	Coverage    Window            `json:"coverage"`
+	LastRun     time.Time         `json:"lastRun"`
+	NextRun     time.Time         `json:"NextRun"`
+	Progress    float64           `json:"progress"`
+	RefreshRate string            `json:"refreshRate"`
+	Resolution  string            `json:"resolution"`
+	StartTime   time.Time         `json:"startTime"`
+}
+
+// ReconciliationStatus describes Reconcilation metadata of a CloudStore
+type ReconciliationStatus struct {
+	Coverage    Window            `json:"coverage"`
+	LastRun     time.Time         `json:"lastRun"`
+	NextRun     time.Time         `json:"NextRun"`
+	Progress    float64           `json:"progress"`
+	RefreshRate string            `json:"refreshRate"`
+	Resolution  string            `json:"resolution"`
```

Comment:
Why is `Resolution` a string and not a `time.Duration`? Looks like this is a pattern, so probably a question for @nikovacevic

Reply:
It's a string in the status struct so that it's human-readable. Otherwise, it just looks like an integer, which is hard to interpret in status responses. It's a `time.Duration` in the ETLs, themselves.

Reply:
Ah, gotcha! Thanks.